linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_disable_nullok/oval/shared.xml

```diff
@@ -0,0 +1,22 @@
+<def-group>
+  <definition class="compliance" id="accounts_passwords_pam_disable_nullok" version="4">
+    {{{ oval_metadata("Blank or null passwords are disallowed in the PAM password-auth file.") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_passwords_pam_disable_nullok"
+      comment="nullok not present in password-auth" negate="true"/>
+    </criteria>
+  </definition>
+
```

Comment:
The main problem for me is that this rule doesn't contain any test scenarios. Please add test scenarios covering most of the situations in the PAM file.